workloads/logging/env.sh

```diff
@@ -13,6 +13,7 @@ export DURATION=${DURATION:-1}
 export MESSAGES_PER_SECOND=${MESSAGES_PER_SECOND:-0}
 export POD_COUNT=${POD_COUNT:-1}
 export TIMEOUT=${TIMEOUT:-600}
+export DEBUG=${DEBUG:-true}
```

Comment:
We normally don't enable DEBUG mode by default

Reply:
changed